crates/turborepo-lib/src/turbo_json.rs

```diff
@@ -0,0 +1,31 @@
+use std::{fs, path::Path};
+
+use anyhow::Result;
+use semver::{Version, VersionReq};
+use serde::Deserialize;
+
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub struct TurboJson {
+    pub turbo_version: String,
+}
+
+pub fn read_turbo_json(path: &Path) -> Result<TurboJson> {
+    let turbo_json_string = fs::read_to_string(path)?;
+    let turbo_json = serde_json::from_str(&turbo_json_string)?;
+
+    Ok(turbo_json)
+}
+
+impl TurboJson {
+    pub fn check_version(&self, version: &str) -> Result<bool> {
+        let version = Version::parse(version)?;
+
+        if self.turbo_version.is_empty() {
```

Comment:
```suggestion
        // version is not required
        if self.turbo_version.is_empty() {
```

Reply:
This line of code gets dropped in the version that it will be rebased on top of:

https://github.com/nathanhammond/turbo/pull/8/files#diff-3c4e635078eed173e0c6dafe5eb719efc6d1d9e2777f09fc95c4c6a826baf231